### PR TITLE
 feat(settings): Expand Settings page functionality 

### LINKED ISSUE #15917: Translations are no longer working
I can't seem to find any page on beta that has a Spanish translation.

### LINKED ISSUE #16237: Add isNewDataVisCert property
#### Issue Description
On the user object, we will need both a `isDataVisCert` and an `isNewDataVisCert` property.

`isDataVisCert` should refer to what we will call the "Legacy Data Visualization Certificate" and is `isNewDataVisCert` should refer to the "Data Visualization Certificate".

### LINKED ISSUE #16271: Social Media Buttons - Unable to remove account(s) - account not found
#### Challenge Name

My Profile -> Update my Settings 
Add my twitter account and add my linked in account

#### Issue Description

added my twitter account to my profile. when trying to remove the account, it gives an error saying no account found. I have not tried for linked in.

#### Browser Information

* Browser Name, Version:  
* Operating System: FireFox 57.0 (64-bit)  and Chrome Version 63.0.3239.84 (Official Build) (64-bit)
* Mobile, Desktop, or Tablet: Laptop Windows 10 Professional 64-

#### Your Code

```js



```
#### Screenshot

image

image

### LINKED ISSUE #16278: Available For Hire does not update the public profile if they are/not available
#### Challenge Name

Profile - Update My Settings - Available for Hire

#### Issue Description

When enabling the 'Available for Hire' it does not appear in the public profile 
that the camper is available for hire.
Maybe add a "Available For Hire: Yes" or "Available For Hire: Yes" depending on the camper's selection
Maybe just above the brownie/cookie points value

#### Browser Information

* Browser Name, Version: 
* Operating System: 
* Mobile, Desktop, or Tablet: 

#### Your Code

```js



```
#### Screenshot

image

### LINKED ISSUE #16279: Language Option menu does not close and turns green when mouse is moved away
#### Challenge Name

My Profile - Update My Settings - Display Challenges in

#### Issue Description

When leaving the dropdown menu open and move the mouse away from the drop down
the menu does not close and it turns green

#### Browser Information

* Browser Name, Version:  
* Operating System: FireFox 57.0 (64-bit)  and Chrome Version 63.0.3239.84 (Official Build) (64-bit)
* Mobile, Desktop, or Tablet: Laptop Windows 10 Professional 64-bit

#### Your Code

```js



```
#### Screenshot

image

### LINKED ISSUE #16285: Changes to privacy setting does not show popup confirm changes
#### Challenge Name

My Profile -> Update my Settings -> Privacy Settings

#### Issue Description

When toggling privacy settings between On or Off, there is no confirmation the changes were done. When you verify your public profile the changes do take effect.

Production FCC does alert when the changes are executed.
The screenshot below is from Production FCC. 
#### Browser Information

Browser Name, Version:
Operating System: FireFox 57.0 (64-bit) and Chrome Version 63.0.3239.84 (Official Build) (64-bit)
Mobile, Desktop, or Tablet: Laptop Windows 10 Professional 64-bit

#### Your Code

```js



```
#### Screenshot

Production FCC:
image

### LINKED ISSUE #16587: [Beta] In Firefox setting page can't align properly with navigation bar.
This alignment problem happen only in setting page. Its happen when browser zoom set in more than 40% and current monitor resolution set in 1920 * 1080 (16:9). But in 40% zoom setting page is harder to read. For more details please check below screenshots. 
### Set in 40% zoom
40 _zoom
### Set in 100% zoom
100 _zoom

Current Environment: 
* Browser Name, Version:  Firefox 58
* Operating System: Linux Mint 18.3
* Mobile, Desktop, or Tablet: Desktop with 21.5" Monitor

### LINKED ISSUE #16597: Resequence bio edit fields
cursor_and_settings___freecodecamp

### LINKED ISSUE #16599: Update email settings
cursor_and_settings___freecodecamp

### LINKED ISSUE #16600: Remove "available for hire" section of settings

### PR DESCRIPTION
**Pull request by Bouncey**
This is part 1/2 re #16412 

Part 2 will be the reactification of the Profile page.

closes #15917
closes #16237
closes #16271
closes #16278
closes #16279
closes #16285
closes #16587
closes #16597
closes #16599
closes #16600

### CODE REVIEW

File: common/app/redux/fetch-challenges-epic.js [hunk 1]
```diff
@@ -54,11 +54,14 @@ export function fetchChallengesEpic(
   { getState },
   { services }
 ) {
-  return actions::ofType(types.appMounted)
+  return actions.do(console.log)::ofType(
```
Comment (BerkeleyTrue) [inline common/app/redux/fetch-challenges-epic.js]: you must always bind the log function to the console object

Comment (Bouncey) [reply common/app/redux/fetch-challenges-epic.js]: That shouldn't be there. I was using it see what was going on

**Comment by QuincyLarson** [comment]
OK - I enjoyed tinkering with this. Here's my feedback (in screenshots for simplicity).

settings___freecodecamp

settings___freecodecamp

camper_ossia_s_code_portfolio___freecodecamp

settings___freecodecamp

camper_ossia_s_code_portfolio___freecodecamp

settings___freecodecamp

settings___freecodecamp

settings___freecodecamp

settings___freecodecamp

settings___freecodecamp

settings___freecodecamp

**Comment by Bouncey** [comment]
This comments on the profile page are out of scope for this PR.

This PR focuses only on the settings page, a subsequent PR will address the Profile page

**Comment by QuincyLarson** [comment]
@Bouncey LGTM! Thanks for addressing my many feedback items.

**Comment by Bouncey** [comment]
@BerkeleyTrue There have been logic changes in the saving of projects and certificate claiming since your last review. Would you like to review again?

**Review by BerkeleyTrue — APPROVED** [review]

**Comment by QuincyLarson** [comment]
@Bouncey Merged! Congratulations on shipping a considerable amount of functionality here!

**Pull request merged**